go.mod

```diff
@@ -2,6 +2,18 @@ module knative.dev/func
 
 go 1.20
 
+replace (
+	// Use older version since it checks ~/.docker/config.json for credentials as opposed to later versions
+	github.com/containers/image/v5 => github.com/containers/image/v5 v5.23.1
+
+	// Tekton CLI uses newer version without currently used constructor code
+	github.com/hinshun/vt10x => github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
```

Comment:
Could this be removed too @dsimansk (provided the code compiles)?

Reply:
Definitely, I recall the library was used in a test setup, mock terminal or something like that. That might have been refactored since then.

Reply:
E.g. here: https://github.com/knative/func/blob/main/cmd/prompt/prompt_test.go#L27. 

Some of the later revision don't have such constructor function.